test/bm.v3

```diff
@@ -0,0 +1,108 @@
+//@execute =1
```

Comment:
This should probably live in `test/large`

Reply:
Oops I was using this to track down a bug and accidentally committed it. Will come up with something smaller that reproduces the bug and add it into test/range!